tests/test_modeling_common.py

```diff
@@ -550,10 +550,21 @@ def check_hidden_states_output(inputs_dict, config, model_class):
             else:
                 seq_length = self.model_tester.seq_length
 
-            self.assertListEqual(
-                list(hidden_states[0].shape[-2:]),
-                [seq_length, self.model_tester.hidden_size],
-            )
+            if "modeling_squeezebert" in str(model_class):
```

Comment:
Hmm should we maybe transpose the `hidden_states` in SqueezeBert before adding it to the list of hidden states. It might be nicer to have all hidden states in the same format and would prevent this hack here. What do you think @LysandreJik ?

Reply:
I agree with this, we don't want to surprise users of the library.